src/main/java/javaserver/ResponseBuilder.java

```diff
@@ -0,0 +1,27 @@
+package javaserver;
+
+public class ResponseBuilder {
```

Comment:
I like that you've made a Builder for your Response, but I think it's strange that they're separated. When using a Builder, you typically put the Builder inside the object as a static inner class and we make the object's constructor private. You may also want to consider what is a required property of a Response and include it inside the constructor of the Builder.